.vib/redis-cluster/goss/goss.yaml

```
@@ -0,0 +1,39 @@
+command:
+  redis-version:
+    exec: redis-server --version | grep --quiet "$APP_VERSION"
+    exit-status: 0
+    stdout: [ ]
+    stderr: [ ]
+    timeout: 20000
+  redis-auth:
+    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h 127.0.0.1 -p 6379
```

Use a dynamic port instead, based on the environment variable that configures it:
```suggestion
    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h 127.0.0.1 -p $REDIS_PORT
```